nanoDFT/experimental_pmap_nanoDFT.py

```diff
@@ -167,16 +170,16 @@ def init_dft_tensors_cpu(mol, opts, DIIS_iters=9):
     L_inv           = np.linalg.inv(np.linalg.cholesky(O))          # (N,N)
     #if opts.backend != "ipu": ERI = mol.intor("int2e_sph")          # (N,N,N,N)=(66,66,66,66) for C6H6.
     #else:                     ERI = None # will be computed on device
-    ERI = mol.intor("int2e_sph")
+    ERI = 0 #mol.intor("int2e_sph")
```

Comment:
Is this intended / temporary? If this is the case, then it might help to add a comment (e.g. "TODO fix later") to find occurrences easily later as several variables are zeroed out at the moment

Reply:
Added TODO.